app/views/casework/send_decisions/new.html.erb

```diff
@@ -26,16 +28,27 @@
     <% if @decisions.all? &:complete? %>
       <%= form_with model: @form_object, url: crime_application_send_decisions_path do |f| %>
 
-        <%= f.govuk_radio_buttons_fieldset :next_step, legend: { tag: 'h2', size: 'm' } do %>
-
-          <%= f.govuk_radio_button :next_step, :send_to_provider, link_errors: true %>
-          <%= f.govuk_radio_button :next_step, :add_another %>
+        <% if @crime_application.work_stream == 'non_means_tested' %>
+          <%= f.govuk_radio_buttons_fieldset :next_step, legend: { hidden: true }, hidden: true do %>
```

Comment:
Would you mind changing this to use a hidden_field input rather than hiding these with css? (e.g. https://api.rubyonrails.org/v7.0/classes/ActionView/Helpers/FormHelper.html#method-i-hidden_field )